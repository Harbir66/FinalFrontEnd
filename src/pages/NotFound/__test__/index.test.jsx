import React from 'react';
import { render } from '@testing-library/react';
import PageNotFound from '../index';

describe('Not Found Page', () => {
  it('should render', () => {
    const { asFragment } = render(<PageNotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
