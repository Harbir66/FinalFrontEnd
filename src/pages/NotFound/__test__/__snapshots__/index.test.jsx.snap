// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page should render 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <div
      class="sidebar"
    >
      <div
        class="sidebar-header"
      >
        CMS+
      </div>
      <div
        class="sidebar-search"
      >
        <span>
          COLLECTION TYPES
        </span>
        <img
          alt="searchIcon"
          src="icon-search-dark@3x.png"
        />
      </div>
      <nav>
        <ul />
      </nav>
      <button
        style="background: black;"
        type="button"
      >
        CONTENT TYPE BUILDER
      </button>
    </div>
    <div
      class="pageNotFoundContainer"
    >
      <header
        class="header"
      >
        Page Not Found
      </header>
      <div
        class="content"
      >
        <p>
          404 Error. Page not found
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
